### PR TITLE
Create remove books

### PR DESCRIPTION
**Pull request by initseis**
In this branch I:

- Used Redux duck-pattern
- Setup the books reducer
- Setup the configureStore
- Used redux-logger
- Update components to add and remove books

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -19,10 +17,6 @@ const Form = () => {
     setNewAuthor(event.target.value);
   };
 
-  const getCategory = (event) => {
-    setCategory(event.target.value);
-  };
-
   const generateId = () => {
```
Comment (bellom) [inline src/components/Form.js]: ### [Optional]

Great job creating this function but you could have simply use [npm uuid](https://www.npmjs.com/package/uuidv4)

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @sevinchek  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Let's make your repo looks professional by updating your `README` file**.
Checkout out the below link for help, feel free to follow this [Microverse template](https://github.com/microverseinc/readme-template) from pathwright

**Kindly check my comments under this review;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

**Review by Emmyn5600 — APPROVED** [review]
Hi, 

Good job so far! 👏🏼 @sevinchek 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSE
 
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by initseis** [comment]
Thanks.

**Pull request merged**